### ISSUE TITLE
Autosave drafts when navigating away from Compose / Reply page

### ISSUE THREAD

**Issue by webinista** [department-of-veterans-affairs/messaging-fe#114]

**Comment by eli-lewis** [comment]
@ayaleloehr I think this is a good idea... move forward?

**Comment by ayaleloehr** [comment]
I don't think we should prioritize this yet. I believe there is still some scoping that needs to be done before we're ready to implement, especially since we can't currently save drafts with attachments, which might cause a strange UX if we're saving drafts automatically without them, but not telling the user that in some way.

**Comment by mikeusds** [comment]
@eli-lewis Let's put this on the agenda when we connect.
